enterprise-beans/4.0/_index.md

```diff
@@ -33,20 +33,20 @@ Jakarta Enterprise Beans defines an architecture for the development and deploym
 
 [Jakarta Enterprise Bean 4.0 Release Record and Plan]()
```

Comment:
I know this wasn't part of your effort, but could you also include a link to the release record?
https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan

Thank you!